brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Off-Canvas - Should open faster.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/010e37c0-d7d0-4271-988d-03a96b600d4d?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Output - https://share.getcloudapp.com/mXuYqzxk

### Types of changes




- Improvement

### How has this been tested?

- Go to the admin dashboard and click on the off-canvas menu.
- Check it is opening faster or not.

### Checklist:
